test/e2e/kubectl.go

```diff
@@ -473,6 +474,25 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 		})
 	})
 
+	framework.KubeDescribe("Kubectl should be able to talk to api server", func() {
+		It("kubectl runing in a pod should talk to api server [Conformance]", func() {
```

Comment:
Because it's hostPath mounting the definition of conformance becomes tricky - is it required that a cluster install /usr/local/bin/kubectl (even distros that don't have that dir?).  @ncdc is there any other place where we shove kubectl into a running pod that we could leverage here?